sdk/src/camera_factory.cpp

```diff
@@ -47,9 +48,10 @@ CameraFactory::buildCamera(std::unique_ptr<DeviceInterface> device) {
 
     case SensorType::SENSOR_96TOF1:
         return std::unique_ptr<Camera>(new Camera96Tof1(std::move(device)));
-
+#ifdef CHICONY_006
```

Comment:
This will result in a warning that says the enumeration value `SensorType::SENSOR_CHICONY` is not handled in the switch case. The #ifdef can be added inside the `case SensorType::SENSOR_CHICHONY` with a `return nullptr` after it.

Reply:
Done